CONTRIBUTING.md

```diff
@@ -61,6 +59,24 @@ TODO: Code Style Guide
 
 ### Issues
 
+1. Check for Existing Issues
+    - Before opening a new issue, please check if a similar issue already exists. You can search through the [existing issues](https://github.com/DSACMS/repodive-tools/issues) to avoid duplicates.
+2. Choose the Correct Issue Type
+    - Bug Report: For reporting bugs or unexpected behavior in the application.
+    - Feature Request: To suggest new features or enhancements.
+    - Question/Help: If you need help understanding a feature or resolving a problem.
+3. Provide Clear Details:
+    - Summary: A clear and concise title.
+    - Description: Explain the problem or feature in detail, including any relevant context.
+    - Steps to Reproduce: If you're reporting a bug, describe the steps someone can follow to reproduce the problem.
+    - Expected Behavior: What did you expect to happen?
+    - Actual Behavior: What actually happened?
+    - Screenshots or Logs: Include any logs, error messages, or screenshots that can help diagnose the issue.
+    - Environment: Mention the environment where the issue occurred (OS, Python version, dependencies).
+4. Labels and Templates
+    - If available, select the appropriate labels (e.g., bug, enhancement, help wanted).
+
+
 <!---
 TODO: Example Issue Guides
 
```

Comment:
Policy section below is outdated, please update to the following: https://github.com/DSACMS/repo-scaffolder/blob/main/tier1/%7B%7Bcookiecutter.project_slug%7D%7D/CONTRIBUTING.md#policies

Reply:
This will need to be updated in upstream repo-scaffolder then, yeah.

Reply:
i believe this is outdated because we were using an old version of the md files that were already in the repo. upstream is updated properly already!